tests/ui/associated-types/defaults-unsound-62211-1.next.stderr

```diff
@@ -6,7 +6,7 @@ LL |     drop(origin);
    |          |
    |          argument has type `<T as UncheckedCopy>::Output`
    |
-   = note: use `let _ = ...` to ignore the expression or result
+   = note: use `let _ = origin` to ignore the value instead of dropping directly
```

Comment:
`let _ = expr;` still drops the expression directly.

Reply:
> `let _ = expr;` still drops the expression directly.

Done. Thank you.